operations/conman/Establish_a_Serial_Connection_to_NCNs.md

```diff
@@ -1,120 +1,148 @@
 # Establish a Serial Connection to NCNs
 
 The ConMan pod can be used to establish a serial console connection with each non-compute node (NCN) in the system.
-In the scenario of a power down or reboot of an NCN worker, one must first determine if any `cray-console-node` pods
-are running on that NCN. It is important to move `cray-console-node` pods to other worker nodes before rebooting or
+
+In the scenario of a power down or reboot of an NCN worker, one must first determine if any `cray-console` pods
+are running on that NCN. It is important to move `cray-console` pods to other worker nodes before rebooting or
```

Comment:
At the end of this sentence you may want to add "... powering off a worker node to minimize disruption in console logging."